### PR TITLE
refactor(cli,ironfish): Rename multisig RPC / CLI

### PR DESCRIPTION
**Pull request by rohanjadvani**
## Summary

UX changes as discussed for multisig

## Testing Plan

Existing tests

## Documentation

Does this change require any updates to the Iron Fish Docs (ex. [the RPC API
Reference](https://ironfish.network/docs/onboarding/rpc/chain))? If yes, link a
related documentation pull request for the website.

```
[ ] Yes
```

## Breaking Change

Is this a breaking change? If yes, add notes below on why this is breaking and
what additional work is required, if any.

```
[ ] Yes
```

### CODE REVIEW

File: ironfish-cli/src/commands/wallet/multisig/participant/create.ts [hunk 1]
```diff
@@ -7,7 +7,7 @@ import { IronfishCommand } from '../../../../command'
 import { RemoteFlags } from '../../../../flags'
 
 export class MultisigIdentityCreate extends IronfishCommand {
-  static description = `Create a multisig identity`
+  static description = `Create a multisig participant identity`
```
Comment (hughy) [inline ironfish-cli/src/commands/wallet/multisig/participant/create.ts:10]: There are a couple of other places in the help text that we refer to `identity`. Should we change those to `participant identity`, too?

File: ironfish-cli/src/commands/wallet/multisig/participants/index.ts [hunk 2]
```diff
@@ -5,7 +5,7 @@ import { Flags } from '@oclif/core'
 import { IronfishCommand } from '../../../../command'
 import { RemoteFlags } from '../../../../flags'
 
-export class MultisigAccountIdentities extends IronfishCommand {
+export class MultisigAccountParticipants extends IronfishCommand {
   static description = `List the identities for a multisig account`
```
Comment (hughy) [inline ironfish-cli/src/commands/wallet/multisig/participants/index.ts:9]: Should we document this as 'participant identities'?

**Review by hughy — APPROVED** [review]
a couple of minor comments on how we refer to 'identity' or 'participant identity' in help text

File: ironfish-cli/src/commands/wallet/multisig/participant/create.ts (hunk 1 above)
Comment (rohanjadvani) [reply ironfish-cli/src/commands/wallet/multisig/participant/create.ts:10]: Yep, updated

File: ironfish-cli/src/commands/wallet/multisig/participants/index.ts (hunk 2 above)
Comment (rohanjadvani) [reply ironfish-cli/src/commands/wallet/multisig/participants/index.ts]: Updated

**Pull request merged**